linoleum89/anti-vialidad
# Login Component

**CHANGELOG**

- Implement a Login component with validation
- Did some cleanup
- Added test for login component
- Added a utils file, for now it only contains a method to retrieve the user data

**TODO**

- After the user is logged, implement a switch route to the reports dashboard and then fetch the reports.

**HOW TO TEST**

- Run `npm install`
- Run `npm start`
- test data: **username**: linoleum89, **password**: linoleum
- If the user is invalid, the form should display an error message:

<img width="1045" alt="screen shot 2019-03-07 at 4 32 37 pm" src="https://user-images.githubusercontent.com/2753274/53996696-a66ffb80-40f6-11e9-9e24-c4c3750d82c1.png">

- Run the unit tests: `npm run test`

![screen shot 2019-03-07 at 4 23 04 pm](https://user-images.githubusercontent.com/2753274/53996717-b7207180-40f6-11e9-9d8c-f3a404afcfe1.png)
